css/reseter.css

```diff
@@ -163,6 +163,7 @@ dl,
 ol,
 ul {
   margin: 1em 0;
+  list-style: none;
```

Comment:
This seems a bit opinionated, can you make the changes in the opinionated scss file instead 🙂